Formula/portable-ruby@2.2.rb

```diff
@@ -64,8 +89,8 @@ def install
     args << "--with-opt-dir=#{paths.join(":")}"
 
     system "./configure", *args
-    system "make"
-    system "make", "install"
+    make
+    make "install"
```

Comment:
In my experience, 99% of things work on make 3.80; only a few things run into its bugs.